### DISCUSSION TITLE
Technical documentation and diagram missing

### DISCUSSION THREAD

**Discussion by HeyBaldur** [dotnet/eShop#373] [Q&A]
Hi guys, 

As usual, thanks for the project and I appreciate it the time and effort, however I am wondering, is there any diagram some technical documentation or something that can give devs an idea what actually is happening? 

I read the code but unfortunately I don't understand anything in terms of the flow of the app. I believe we need a bit more info. And if the documentation is not available, is it possible to add a note in the readme or somewhere visible that indicates the documentation is not available for now, also I took time to investigate if there is somewhere hidden but no luck. 

I appreciate it in advance.

**Comment by OlehObodinWork** [comment]
That would be great for educational purposes. So, I am on the waitlist.

**Comment by rlarno** [comment]
See this discussion item: https://github.com/dotnet/eShop/discussions/262
